README.md

```diff
@@ -29,6 +29,10 @@ remoteagent1-dcos112s.example.com
 [agents_public]
 publicagent1-dcos112s.example.com
 
+[win_agents]
+agent1-windows.example.com
```

Comment:
`agent1-windows.example.com ansible_user=Administrator ansible_password=<mysecurepassword>`

Reply:
added to inventory example